js/tools/gradientBrush.js

```diff
@@ -0,0 +1,50 @@
+window.tools.gradientBrush = {
+  name: 'gradientBrush',
+  icon: '/images/gradient-icon.png',
+  state: {
+    selected: false,
+    mousePressed: false,
+    prevMouse: { x: null, y: null },
+    baseColor: { g: 150, b: 150 }
+  },
+  events: {
+    mousedown: function (e, self) {
+      self.state.mousePressed = true
+    },
+    mouseup: function (e, self) {
+      self.state.mousePressed = false
+      self.state.prevMouse = { x: null, y: null }
+      self.state.baseColor = getRandomColor();
```

Comment:
do u have the StandardJS plugin installed/working? if so it should have warned u that this is an unnecessary `;` at the end of this line

Reply:
Hmmm I realised I have 2 vscodes, I was using the one without it the plugin